Makefile

```diff
@@ -115,10 +115,10 @@ clean:
 	@rm -f coverage*.out
 
 .PHONY: build
-build: generate fmt vet lint _build
+build: generate lint _build
```

Comment:
I wonder if we could also drop the `lint` from here since that's done separately anyway in https://github.com/Kong/kubernetes-ingress-controller/blob/e9167e2c32bea34a2e8ad78314c4e099f35b0c9c/.github/workflows/_linters.yaml

Reply:
I think yes, `make build` is only referenced in `CONTRIBUTING.md`, which does not include information about linting. I'm adding it